prerequisites-check.sh

```diff
@@ -13,7 +16,12 @@ check_command_present git
 check_command_present docker
 
 YQ_VERSION="$(yq --version | ${SED} 's/^.* //g')"
+# After version 3.3.1, yq --version sends the string to STDERR instead of STDOUT
+if [ -z "$YQ_VERSION" ]; then
+    YQ_VERSION="$(yq --version 2> >(grep -Po '\d.\d.\d'))"
```

Comment:
Thsi should probably use the GREP from the multi-platform-support.

Reply:
Good point.